app/assets/stylesheets/style.css.scss

```diff
@@ -1,6 +1,43 @@
 @import url(https://fonts.googleapis.com/css?family=Pacifico);
 @import url(https://fonts.googleapis.com/css?family=Lato:300,400);
 @import 'foundation-icons';
+@import "font-awesome-sprockets";
+@import "font-awesome";
+
+.fa-star-o {
+  font-size: 2em;
+}
+
+.fa-star {
+  font-size: 2em;
+  color: #41A5B2;
+}
+
+.button {
+  font-family: Lato;
+  background-color: #41A5B2;
+}
+.row .row {
+  width: 100%;
+}
+
+.top-bar .left .fi-magnifying-glass {
+  font-size: 2.5em;
+  color: #FFFFFF;
+}
+
+.top-bar .left #search-box .button {
+  background-color: #41A5B2;
```

Comment:
Color literals like `#41A5B2` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#41A5B2` should be written as `#41a5b2`